reinforcement_learning/rl_resource_allocation_ray_customEnv/rl_bin_packing_ray_custom.ipynb

```diff
@@ -191,6 +191,30 @@
     "    !/bin/bash ./common/setup.sh"
```

Comment:
<p>This is not a custom image. We should just let the RLEstimator pointing to this image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Right, fixed.